notebooks/run_model_on_snellius.ipynb

```diff
@@ -0,0 +1,300 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #23.        # create an output file
</pre><p>we dont need to create the log file as the model output will appear in the notebook by itself. So, we can remove this part.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/EcoExtreML/processing/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>